### PR TITLE
Like-reaction-alignment&share-reaction-alignment

### PR DESCRIPTION
**Pull request by ramandeepR**
Like-reaction-alignment done

### CODE REVIEW

File: src/components/like-button.vue [hunk 1]
```diff
@@ -60,4 +60,19 @@ export default defineComponent({
...
 });
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+
+ion-chip span{
+  margin-top:2px;
+  padding-right: 6px !important;
```
Comment (gnunicorn) [inline src/components/like-button.vue:69]: Any way we can get around using `!important`? in my experience this always bites one in the butt later.

File: src/components/reactions.vue [hunk 2]
```diff
@@ -102,5 +102,22 @@ export default defineComponent({
...
+  margin: 4px;
+  padding: 7px 1px;
+  display: inline-flex;
+  position: relative;
+  align-items: center;
+  height: 32px;
+  background: var(--background);
+  color: var(--color);
+  font-family: var(--ion-font-family, inherit);
+  font-size: 14px;
```
Comment (gnunicorn) [inline src/components/reactions.vue:116]: Are all of them really necessary? And are they responsive? `ionic` already provides a lot of helpers and sizes, using `px` here almost certainly means there will be issues with other sizes.

File: src/components/share-button.vue [hunk 3]
```diff
@@ -91,4 +91,18 @@ export default defineComponent({
...
   },
 });
-</script>
\ No newline at end of file
+</script>
+<style scoped>
+
+ion-chip span{
+  margin-top:2px;
+  padding-right: 6px !important;
```
Comment (gnunicorn) [inline src/components/share-button.vue:99]: same as before

**Review by gnunicorn — CHANGES_REQUESTED** [review]
As this is primarily a UI bug could you please post screenshots of the result? Please using responsive mode for iPhone (6 or), iPad, iPad Landscape and Desktop - so the results can be reviewed as complete and responsive?

File: src/components/like-button.vue (hunk 1 above)
Comment (ramandeepR) [reply src/components/like-button.vue:69]: This is for make sure no other same class overrides its property padding-right. As same classes used onside other ion-chips.

File: src/components/reactions.vue (hunk 2 above)
Comment (ramandeepR) [reply src/components/reactions.vue:116]: Yes, define properties about the class which I have used. This is not used as root, so using `px` not effect the sizes on others too

File: src/components/share-button.vue (hunk 3 above)
Comment (ramandeepR) [reply src/components/share-button.vue:99]: same as before

**Comment by ramandeepR** [comment]
> As this is primarily a UI bug could you please post screenshots of the result? Please using responsive mode for iPhone (6 or), iPad, iPad Landscape and Desktop - so the results can be reviewed as complete and responsive?

desktop-version
iPhone 6_7_8_landscape
iPhone 6_7_8_portrait

**Comment by gnunicorn** [comment]
I think there's a misunderstanding with the plus-button - it is not meant to be that prominent and outlined (unless you hover). The alignment might needs to be adapted but the rest of the design isn't meant to be changed (border and corner and all) - it should visually be the same as the comment-button

File: src/components/reactions.vue (hunk 2 above)
Comment (gnunicorn) [reply src/components/reactions.vue:116]: I don't mean other elements, mainly other font-sizes, I mean if were to change the theme of ionic. I don't see why we'd need to set color, font-family, font-size here at all, these should all be inherited from button (same of cursor-pointer). Please go through these add remove everything that isn't strictly necessary to get the alignment fixed - we should only change elements minimally on this lowest level to not mess with broader ionic theming facilities.

**Review by gnunicorn — APPROVED** [review]
Thanks for the screenshots. Once the remarks about the button are addressed, we are ready to merge.

File: src/components/reactions.vue (hunk 2 above)
Comment (ramandeepR) [reply src/components/reactions.vue:116]: I have removed the other elements

**Pull request merged**